tests/unit/test___init__.py

```diff
@@ -1,6 +1,6 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
-
+# pylint: disable=C0302
```

Comment:
What is this disable please? I think pylint let's you use the name which makes it easier to figure out what is being disabled. Also, we have a standard for including a note explaining why the disable is needed please

Reply:
This is somewhat related to the previous comment, where we have classes/files becoming to complex (hence the check disabled). We also have an issue about this https://github.com/canonical/upload-charm-docs/issues/175

BTW...I have changed from code to names